monkey/infection_monkey/post_breach/pba.py

```diff
@@ -46,8 +47,21 @@ def run(self):
         """
         exec_funct = self._execute_default
         result = exec_funct()
+        if self.scripts_were_used(result):
+            T1064Telem(ScanStatus.USED, "Scripts used to execute %s post breach action." % self.name).send()
         PostBreachTelem(self, result).send()
 
+    def scripts_were_used(self, pba_execution_result):
+        """
+        Determines if scripts were used to execute PBA
+        :param pba_execution_result: result of execution function. e.g. self._execute_default
+        :return: True if scripts were used, False otherwise
+        """
+        pba_execution_succeeded = pba_execution_result[1]
+        if pba_execution_succeeded and isinstance(self.command, list) and len(self.command) > 1:
```

Comment:
The following pattern:
```
if cond:
  return True
return False
```
should always be replaced with:
`return cond`

Reply:
God damn it. How do I keep forgetting that....